testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -71,6 +73,16 @@
 @ExtendWith(CqlSessionExtension.class)
 @CqlSessionSpec()
 public class RestApiv2Test extends BaseIntegrationTest {
+  // Due to Guava's null hatred, ImmutableMap cannot contain nulls.
+  // But we have a need for null values in JSON so here's something that
+  // will get serialized as JSON null.
+  private final Object GUAVA_NULL_VALUE =
```

Comment:
As per comment, Guava barfs of `null`s as not just Map keys but values. But we can relatively easily use a "token" that serializes (via Jackson) as `null`; this is one possible way to achieve that.